### PR TITLE
Use mainnet url in client

### PR DESCRIPTION
**Pull request by asutula**

### CODE REVIEW

File: pkg/client/chains.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 )
 
 const (
+	mainnetURL = "https://tableland.network"
 	testnetURL = "https://testnet.tableland.network"
```
Comment (asutula) [inline pkg/client/chains.go:9]: Just want to be sure... we need to update the testnet url at some point in the future, correct? Or did we find a way to let the existing testnet url remain the same?

Comment (jsign) [reply pkg/client/chains.go:9]: See [here](https://www.notion.so/textile/Tableland-Testnet-Mainnet-separation-68c67911e68b410291cc03c911b97655#e647b9d032ed4c2b8d21f1976d03ebfe).

**Comment by jsign** [comment]
> _ No description provided. _

Do you mind targeting this PR to [this branch](https://github.com/tablelandnetwork/go-tableland/pull/396)?
So whenever we merge that PR and we have the official split, this goes bundled with it.

File: pkg/client/chains.go (hunk 1 above)
Comment (jsign) [reply pkg/client/chains.go:9]: Notice also what's described in that document regarding the testnet network.

Comment (asutula) [reply pkg/client/chains.go:9]: Ok so my interpretation of that is I should update this PR to use `testnets.tableland.network` for the `testnetURL`, correct?

Comment (jsign) [reply pkg/client/chains.go:9]: Yep

**Comment by asutula** [comment]
Ok @jsign if you want to give me a final approval, I'll merge this.

**Review by jsign — APPROVED** [review]

**Pull request merged**